blankiety_galantow/resources/game/index.html

```diff
@@ -81,8 +86,13 @@ <h4>Limit czasu</h4>
                 <div class="card w3-card w3-round-large w3-hover-shadow w3-center hvr-float" 
                     v-for="card in myCards" 
                     v-bind:class="{selected: card.selected}" 
```

Comment:
Skoro mamy fajną tablicę `selectedCards`, to trzymanie w każdej karcie pola `selected` jest redundantne. Polecam wywalić te pola z kart, wyrzucić też linię **16** z pliku *main.js*, a powyższe zastąpić:
```
v-bind:class="{selected: selectedCards.includes(card)}"
```